packages/package-bundler/README.md

```diff
@@ -8,165 +8,52 @@ ESM code is tree-shakeable.
 pnpm add -D @kaizen/package-bundler
 ```
 
-## build-shared-ui
+## Presets
 
-For shared UI packages. CSS modules and/or Tailwind will be included in the dist in a single stylesheet (`{PACKAGE_NAME}/dist/styles.css`) to be imported by the consumer.
+- [Default preset](src/presets/default/README.md)
+- [Shared UI preset](src/presets/shared-ui/README.md)
 
-_Note: If your package extends another shared UI package, you will need to list the other package as a peerDependency and have the end consumer import both stylesheets._
+## Package entrypoints
 
-### `package.json`
+Depending on the compatibility you wish to provide for your consumer, you may need to define your `package.json` differently.
+Read more about [the different types of package.json entrypoints here](https://nodejs.org/api/packages.html#nodejs-packagejson-field-definitions).
 
-Add the following to your `package.json`:
+### Node 12+
 
-```json
-"main": "dist/cjs/index.cjs", // CommonJS entrypoint
-"module": "dist/esm/index.mjs", // ES modules entrypoint
-"types": "dist/types/index.d.ts", // TypeScript types entrypoint
-"files": [
-  "dist" // Ensure dist dir is included in package
-],
-"sideEffects": [
-  "styles.css" // Ensure styles do not get tree-shaken
-],
-"scripts": {
-  "build": "pnpm package-bundler build-shared-ui",
-}
-```
-
-#### Dependencies
-
-```sh
-pnpm add -D postcss postcss-preset-env rollup tslib
-```
-
-`devDependencies`:
-
-- `postcss`
-- `postcss-preset-env`
-- `rollup`
-- `tslib` (peerDep of `rollup`)
-- If using Tailwind:
-  - `tailwindcss`
-
-### Required files
+If your consumers are all using Node 12+ (including for their TypeScript [`moduleResolution`](https://www.typescriptlang.org/tsconfig/#moduleResolution)), you should be able to use the [`exports`](https://nodejs.org/api/packages.html#exports) field if you have multiple entrypoints.
 
-- `postcss.config.js`
-- `rollup.config.mjs`
-- `tsconfig.json`
-- `tsconfig.dist.json`
-- If using Tailwind:
-  - `tailwind.config.js`
-  - `src/tailwind.css`
+### Node <=10
 
-### PostCSS
+If your consumers may be using version 10 or lower, the `exports` field will not be resolved. Your only option will be to utilise `main` (resolves CJS), `module` (resolves ESM), and `types` (resolves TypeScript).
 
-In `postcss.config.js`:
+Should you want multiple entrypoints, you will need to create a directory relative from the root of the package and create a `package.json` within that contains the extra resolutions.
 
-```js
-module.exports = {
-  plugins: {
-    'postcss-import': {},
-    'cssnano': {},
-  },
-}
-```
-
-If using Tailwind:
-
-```js
-module.exports = {
-  plugins: {
-    'postcss-import': {},
-    'tailwindcss/nesting': 'postcss-nesting',
-    'tailwindcss': {},
-    'autoprefixer': {},
-    'cssnano': {},
-  },
-}
-```
-
-### Rollup
+Eg. If we want to create an entrypoint at `@kaizen/package-bundler/future`:
```

Comment:
Do we even let people have anything under Node v10? Aren't there security rules or something we breach if we run on a version this low? 🤔

Reply:
Probably 🤔 But it seems most of our repos use Node10 module resolution (KAIO currently included because we want to ensure we catch issues which our consumers might be seeing, even though it would make the most sense for us to use `bundler`). Unless everyone moves away from that, this is still relevant 🤷🏻

I suppose technically it's just an inaccurate TypeScript setting? 😂 Might be okay to remove this if we can get everyone using `bundler`?

What do you reckon would make the most sense?

Reply:
I think with the changes it's fine - it just wasn't clear what was meant by 'Node10' - but it's been updated to include the specific `moduleResolution` names now. 👍